### PR TITLE
KEYCLOAK-6775 : fix french typos

### PR DESCRIPTION
**Pull request by sebastienblanc**

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties [hunk 1]
```diff
@@ -104,17 +104,17 @@ totpStep2=Ouvrez l''application et scannez le code-barres ou entrez la clef.
 totpStep3=Entrez le code \u00e0 usage unique fourni par l''application et cliquez sur Sauvegarder pour terminer.
 
 totpManualStep2=Ouvrez l''application et entrez la clef
-totpManualStep3=Utilisez les valeurs de configuration suivante si l''application les authorise
+totpManualStep3=Utilisez les valeurs de configuration suivante si l''application les autorise
 totpUnableToScan=Impossible de scanner ?
-totpScanBarcode=Scanner le code bare?
+totpScanBarcode=Scanner le code-barre ?
```
Comment (gaetancollaud) [inline themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties]: Seems like `code-barres` has an `s` => http://www.larousse.fr/dictionnaires/francais/code-barres_codes-barres/16884

**Review by gaetancollaud — CHANGES_REQUESTED** [review]
minor mistake

**Comment by stianst** [comment]
@sebastienblanc bump

**Review by gaetancollaud — APPROVED** [review]

**Comment by sebastienblanc** [comment]
@stianst someone with reiew/merge rights can merge this, I can not do it myself, can you approve the PR ?

**Review by stianst — APPROVED** [review]

**Pull request merged**